core/cautils/localgitrepository.go

```diff
@@ -72,7 +72,10 @@ func (g *LocalGitRepository) GetBranchName() string {
 func (g *LocalGitRepository) GetRemoteUrl() (string, error) {
```

Comment:
Maybe add here a comment describing the different supported remotes as [described in the PR](https://github.com/kubescape/kubescape/pull/1017#issue-1525813301)